### DISCUSSION TITLE
Compatible Versions

### DISCUSSION THREAD

**Discussion by zu1k** [zu1k/sdusrun#8] [General]
Compatible versions based on usage feedback statistics.

- V1.18
  - V1.18 B20180614
  - V1.18 B20181203
  - V1.18 B20190830
  - V1.18 B20201030
  - V1.18 B20210204
  - V1.18 B20210607
  - V1.18 B20210909
  - V1.18 B20220304
  - V1.18 B20220802
  - V1.18 B20220921
  - V1.18 B20230324  (配置 acid=1 后可用)
- v2.0
  - v=2.00.20220802

Reply to add a feedback.

**Comment by hotwa** [comment]
V1.18 B20201030

**Comment by Revincx** [comment]
V1.18 B20180614

**Comment by Heeteve** [comment]
V1.18 B20220304

**Comment by DefensiveSniper** [comment]
v=2.00.20220802

**Comment by geniuszpp** [comment]
V1.18 B20220921

**Comment by vihor3** [comment]
V1.18 B20181203

**Comment by cxw620** [comment]
SRunCGIAuthIntfSvr V1.18 B20220802 可用

**Comment by liqun1024** [comment]
V1.18 B20210204

**Comment by mofeng-git** [comment]
SRunCGIAuthIntfSvr V1.18 B20230324
配置 acid=1 后可用

**Comment by aqquser** [comment]
V1.18 B20231026
登不上
error_msg: "Nas type not found.",

.\srun login -c config.json
will try at most 3 times...
try 1/3: failed
try 2/3: failed
try 3/3: failed
PortalResponse {
    server_flag: 0,
    services_intf_server_ip: "",
    services_intf_server_port: "",
    access_token: "",
    checkout_date: 0,
    ecode: S(
        "",
    ),
    error: "login_error",
    error_msg: "Nas type not found.",
    client_ip: "10.161.40.218",
    online_ip: "10.161.40.218",
    real_name: "",
    remain_flux: 0,
    remain_times: 0,
    res: "login_error",
    srun_ver: "SRunCGIAuthIntfSvr V1.18 B20231026",
    suc_msg: "",
    sysver: "",
    username: "",
    wallet_balance: 0,
    st: 1730965371,
}

**Reply by mofeng-git** [reply]
> V1.18 B20231026
> 登不上
> error_msg: "Nas type not found.",
> 
> .\srun login -c config.json
> will try at most 3 times...
> try 1/3: failed
> try 2/3: failed
> try 3/3: failed
> PortalResponse {
>     server_flag: 0,
>     services_intf_server_ip: "",
>     services_intf_server_port: "",
>     access_token: "",
>     checkout_date: 0,
>     ecode: S(
>         "",
>     ),
>     error: "login_error",
>     error_msg: "Nas type not found.",
>     client_ip: "10.161.40.218",
>     online_ip: "10.161.40.218",
>     real_name: "",
>     remain_flux: 0,
>     remain_times: 0,
>     res: "login_error",
>     srun_ver: "SRunCGIAuthIntfSvr V1.18 B20231026",
>     suc_msg: "",
>     sysver: "",
>     username: "",
>     wallet_balance: 0,
>     st: 1730965371,
> }

可以试试修改配置文件的 acid 值，我此前也遇到了这个报错，改了 acid 之后就可以正常登录了。

**Comment by MooreFoss** [comment]
V1.18 B20190701可用

**Comment by WeiTool** [comment]
V1.18 B20200918

**Comment by guang-zi** [comment]
V1.18 B20240802可用 config.json中配置 acid=1 后可用

**Comment by yundantianchang** [comment]
V1.18 B20231122 在 config.json中配置 acid=8 后可用。

**Comment by xin-521** [comment]
请问怎么查看认证系统版本

**Comment by xin-521** [comment]
202.194.15.87为什么他会去连接山东大学的教育网，我不在山东大学？

**Comment by X-unable** [comment]
ver1.18 B20240902 在config.json中修改acid=1后可以正常使用

**Comment by hui-shao** [comment]
`SRunCGIAuthIntfSvr V1.18 B20260311` 在配置文件设置 `acid=1` 可用
